src/store/modules/accounts/ledger.js

```diff
@@ -27,14 +27,14 @@ export default {
       ...rootGetters['accounts/getByType']('ledger').map(({ source: { idx } }) => idx),
       -1,
     ) + 1,
-    ledgerAppApi: () => new Ae(new TransportU2F()),
   },
 
   actions: process.env.IS_MOBILE_DEVICE ? {
     sign: signOnMobile,
     signTransaction: signOnMobile,
   } : {
-    async request({ getters: { ledgerAppApi }, dispatch }, { name, args }) {
+    async request({ dispatch }, { name, args }) {
+      const ledgerAppApi = new Ae(await TransportWebUSB.create());
```

Comment:
isn't it expencive to `await TransportWebUSB.create()`? You can put it into vuex state on the first call. Also, does it make sense to call `close`? https://www.npmjs.com/package/@ledgerhq/hw-transport-webusb#close